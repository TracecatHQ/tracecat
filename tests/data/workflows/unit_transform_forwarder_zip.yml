title: Forwarder loop with a list of iterables (zip)
description: Test that we can loop
entrypoint: a
inputs:
  first: 1
  second: 2
  list_one: [1, 2, 3]
  list_two: ["one", "two", "three"]

actions:
  - ref: a
    action: core.transform.forward
    for_each:
      - ${{ for var.x in INPUTS.list_one }}
      - ${{ for var.y in INPUTS.list_two }}
    args:
      value: I received ${{ FN.concat(str(var.x),  var.y) }} from you
