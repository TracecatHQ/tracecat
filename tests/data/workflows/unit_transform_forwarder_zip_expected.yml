ACTIONS:
  a:
    result:
      - I received 1one from you
      - I received 2two from you
      - I received 3three from you
    result_typename: "list"

INPUTS:
  first: 1
  second: 2
  list_one: [1, 2, 3]
  list_two: ["one", "two", "three"]
TRIGGER:
